mp/game/momentum/scripts/HudAnimations.txt

```diff
@@ -113,6 +113,19 @@ event FadeOutStageVel
     Animate HudSpeedMeter StageVelAlpha "0" Accel 5.0 0.75
 }
 
+event FirstStickyArm
+{
+    Animate CHudStickybombs FirstStickyColor "MomentumBlue" Accel 0.01 0.55
```

Comment:
Another thing to consider is you should be able to reference CPanelAnimationVars here as the target color as well. So this could be "StickyArmColor" as an animation var reference and lets users definite a different arm color via the res file.

Reply:
I was not able to reference it as the target. Looking at the other animations I think you can only use ones in `SourceScheme.res`, or a literal.